source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,223 @@
+- The following Jenkins plugins should be enabled: [Git](https://wiki.jenkins-ci.org/display/JENKINS/Github+Plugin) , [Github](https://wiki.jenkins-ci.org/display/JENKINS/GitHub+Plugin), [Github Pull Request Builder](https://wiki.jenkins-ci.org/display/JENKINS/GitHub+pull+request+builder+plugin), [Environment Injector](https://wiki.jenkins-ci.org/display/JENKINS/EnvInject+Plugin), [Conditional Build Step](https://wiki.jenkins-ci.org/display/JENKINS/Conditional+BuildStep+Plugin), and [Run Condition](https://wiki.jenkins-ci.org/display/JENKINS/Run+Condition+Plugin).
+- A Drupal site on Pantheon, with [multidev](https://pantheon.io/features/multidev-cloud-environments) enabled.
+- A [GitHub](https://github.com) account.
+
+## Step 1: Local Project instantiation
+
+1. From your local terminal, use Composer to make a new local project based on our example, which contains Drupal 8, behat, and other configuration settings. In this example our project is call called "jenkins-int":
+
+```nohighlight
+SITE="my-site"
+composer create-project pantheon-systems/example-drops-8-composer $SITE
+cd $SITE
+composer prepare-for-pantheon
+```
+
+2. Initialize a local Git repository within your project. 
+
+```nohighlight
+git init
+git add -A .
+git commit -m "Initial commit."
+```
+   
+## Step 2: Create and configure a GitHub Repository
+
+1. Create a new GitHub repository, without creating a readme or .gitignore file.
+
+![New empty repository](/source/docs/assets/images/integrations/jenkins/new_repo.png)
+
+
+2. Next, connect your local project to this repo as "origin," and push the code to the master branch:
+
+```nohighlight
+git remote add origin git@github.com:YOUR-ORG/YOUR-PROJECT.git
+git push -u origin master
+```
+
+At this point, you should be able to create a local branch, commit a change, and push to GitHub. On GitHub, you should then be able to open a pull request successfully.
+
+## Step 3: Create a Pantheon Site
+
+Now we will spin up a Drupal 8 site on Pantheon with Terminus, then overwrite the default install with the code from our GitHub repo.
+
+1. From your local terminus, use terminus to create a site on Pantheon:
+```nohighlight
+terminus site:create $SITE "My Site" "Drupal 8" --org="My Team"
+terminus connection:set $SITE.dev git
+```
+
+2. Add the Pantheon remote repository address and push the code to it.
+```nohighlight
+PANTHEON_REPO=$(terminus connection:info $SITE.dev --field=git_url)
+git remote add pantheon $PANTHEON_REPO
+git push --force pantheon master
+
+```
+
+3. Then complete the installation on Pantheon.
+
+```nohighlight
+terminus build-env:site-install --site-mail="<your email>" --site-name="My Drupal Site" --account-mail="<your email>" --account-name="admin" $SITE.dev
+```
+
+5. Verify the site is installed and working.
+```nohighlight
+terminus dashboard:view $SITE
+```
+
+Now the master branch of GitHub, your local, and Pantheon are in sync.
+
+## Step 4: Create Jenkins Project #1
```

Comment:
Changed. This was a part I missed when I was going to do two projects: one to test the PR and one to test master. Since they projects were similar, the conditional step solves that and makes the guide a bit shorter. Changed the heading accordingly and removed the incorrect instructions.